architecture/opp-architecture.adoc

```diff
@@ -5,6 +5,16 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Discover {product-title} architecture and concepts.
+{product-title} is a single hybrid-cloud platform for enterprises to build, deploy, run, and manage intelligent applications securely across infrastructures. It is based on Red Hat Enterprise Linux, Kubernetes, and Red Hat {ocp}, and includes the following products:
 
-include::modules/opp-architecture-overview.adoc[leveloffset=+1]
\ No newline at end of file
+* Red Hat Advanced Cluster Management for Kubernetes - Controls clusters and applications from a single console.
+* Red Hat Advanced Cluster Security for Kubernetes - Provides information about cluster security, visibility management, and security compliance.
+* Red Hat Quay - A private container registry that stores, builds, and deploys container images.
+* Red Hat OpenShift Data Foundation Essentials - A data and storage services platform that provides a permanent place to store data while clusters spin up and down across environments.
```

Comment:
I would add Red Hat OpenShift Data Foundation to your common-attributes.adoc file. In the OpenShift main branch for common-attributes.adoc file it is listed as 

> `:rh-storage-first: Red Hat OpenShift Data Foundation`

Maybe coordinate with  Kathryn to see if it should be added?  If and when you add this to your common-attributes file just go through the whole doc and make sure you update any hardcoded referenced to "Red Hat OpenShift Data Foundation" references to the attribute reference.